test/server.test.js

```diff
@@ -36,19 +37,26 @@ test('listen should accept stringified number port', t => {
 
 test('listen should reject string port', async (t) => {
   t.plan(2)
-
   const fastify = Fastify()
   t.teardown(fastify.close.bind(fastify))
 
   try {
     await fastify.listen({ port: 'hello-world' })
   } catch (error) {
-    t.same(error.message, 'options.port should be >= 0 and < 65536. Received hello-world.')
+    if (semver.lt(process.version, '19.1.0')) {
```

Comment:
Error message never guarantee to be same in node major version.
Can we check the `code` instead of `message`.

Reply:
Have to check it when i am again at a computer.

Reply:
applied (Y)